src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -582,6 +595,7 @@ private function addAssetsSection(ArrayNodeDefinition $rootNode)
                                         ->end()
                                     ->end()
                                     ->scalarNode('version_format')->defaultNull()->end()
+                                    ->scalarNode('json_manifest_path')->defaultNull()->end()
```

Comment:
It's a good thought. I don't know! It's a fairly easy thing to deprecate and change in the future anyways, and I can't see any other format in the near future. So let's go with what we like best right now. I changed to `json_manifest_path` because I thought it might be more clear what this manifest thing should be.